CHANGELOG.md

```diff
@@ -74,6 +74,10 @@ and this project adheres to
   mechanism to reliably fetch Firecracker PID. With this change, Firecracker
   process's PID will always be available in the Jailer's root directory
   regardless of whether new_pid_ns was set.
+- [#4468](https://github.com/firecracker-microvm/firecracker/pull/4468): Made
+  MMDS return a response including 'Content-Length' header field even if the
+  body is empty. Otherwise, the client may keep waiting for the connection
```

Comment:
nit: I would rewrite as: "Fixed a bug where an MMDS client would hang or timeout when querying for an empty field, because the `Content-Length` response header was missing.

Reply:
thanks! applied your suggestion!